include/mouse.h

```diff
@@ -21,20 +22,88 @@
 
 #include "dosbox.h"
 
-void Mouse_ShowCursor(void);
-void Mouse_HideCursor(void);
+// ***************************************************************************
+// Notifications from external subsystems - all should go via these methods
+// ***************************************************************************
 
-bool Mouse_SetPS2State(bool use);
+void Mouse_EventMoved(int32_t x_rel, int32_t y_rel, int32_t x_abs, int32_t y_abs, bool is_captured);
+void Mouse_EventPressed(uint8_t idx);
+void Mouse_EventReleased(uint8_t idx);
+void Mouse_EventWheel(int32_t w_rel);
+
+void Mouse_SetSensitivity(int32_t sensitivity_x, int32_t sensitivity_y);
+void Mouse_NewScreenParams(uint16_t clip_x, uint16_t clip_y, uint16_t res_x, uint16_t res_y,
+                           bool fullscreen, int32_t x_abs, int32_t y_abs);
+void Mouse_NotifyMovedVMW();
```

Comment:
As per our conventions, can you make all these functions args `const`?

Reply:
It's not quite obvious what `VMW` is, could you name this function better?

Reply:
As per our conventions, these should be called `MOUSE_SetSensitivity`, etc. (module name uppercased)